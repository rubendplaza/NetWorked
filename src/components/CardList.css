.grid-item {
    align-self: center;
}