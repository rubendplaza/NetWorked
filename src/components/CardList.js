import React from "react";
import Card from "./Card";
import "./CardList.css";

const CardList = ({ connections }) => {
  return (
    <div className="tc grid-item">
      {connections.map((user, i) => {
        return (
          <Card
            key={user.index}
            name={`${user.name.first} ${user.name.last}`}
            email={user.email}
            phone={user.phone}
            company={user.company}
          />
        );
      })}
    </div>
  );
};

export default CardList;
