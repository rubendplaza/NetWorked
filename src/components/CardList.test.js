import React from "react";
import CardList from "./CardList";
import { shallow } from "enzyme";

test("expect to render card component", () => {
  const connections = [
    {
      index: 1,
      name: {
        first: "Ruben",
        last: "Plaza",
      },
      email: "ruben@gmail.com",
      phone: 1234567890,
      company: "Google",
    },
    {
      index: 2,
      name: {
        first: "Rube",
        last: "Plaz",
      },
      email: "rube@gmail.com",
      phone: 1234567899,
      company: "Facebook",
    },
  ];
  expect(shallow(<CardList connections={connections} />)).toMatchSnapshot();
});
