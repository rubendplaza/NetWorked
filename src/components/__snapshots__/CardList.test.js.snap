// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
<div
  className="tc grid-item"
>
  <Card
    company="Google"
    email="ruben@gmail.com"
    key="1"
    name="Ruben Plaza"
    phone={1234567890}
  />
  <Card
    company="Facebook"
    email="rube@gmail.com"
    key="2"
    name="Rube Plaz"
    phone={1234567899}
  />
</div>
`;
